### PR TITLE
Update GitHub profile to reflect new logo and project names

### PR DESCRIPTION
**Pull request by fekete-robert**
Adds openclarity logo and updates component names

### CODE REVIEW

**Comment by lelia** [comment]
@Tehsmash / @lgecse could either of you confirm that these project renames are in-line with broader plans to unify OpenClarity? Are we also changing the official casing from `OpenClarity` to `Openclarity` ?

(ref: https://github.com/openclarity/vmclarity/pull/869#pullrequestreview-1704460130)

File: profile/README.md [hunk 1]
```diff
@@ -31,9 +44,9 @@ Key Capabilities:
 - Comprehensive dashboard for SBOM analysis
 - Pluggable architecture
 
-## [APIClarity](https://github.com/openclarity/apiclarity)
+## [API Security](https://github.com/openclarity/apiclarity)
```
Comment (fekete-robert) [inline profile/README.md:47]:
```suggestion
## API Security
```

File: profile/README.md [hunk 2]
```diff
@@ -31,9 +44,9 @@ Key Capabilities:
...
 
-APIClarity is an open source tool for APIs that helps you visualize and identify potential risks around API usage in your cloud native environments. It helps build the OpenAPI specifications for all APIs in your environment, then helps track drift, shadow or zombie usage for those APIs. You can then use this information to build your application security posture.
+API Security is an open source tool for APIs that helps you visualize and identify potential risks around API usage in your cloud native environments. It helps build the OpenAPI specifications for all APIs in your environment, then helps track drift, shadow or zombie usage for those APIs. You can then use this information to build your application security posture.
```
Comment (fekete-robert) [inline profile/README.md:49]:
```suggestion
[APIClarity](https://github.com/openclarity/apiclarity) is an open source tool for APIs that helps you visualize and identify potential risks around API usage in your cloud native environments. It helps build the OpenAPI specifications for all APIs in your environment, then helps track drift, shadow or zombie usage for those APIs. You can then use this information to build your application security posture.
```

**Review by lgecse — APPROVED** [review]

File: profile/README.md [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
 
-OpenClarity is a suite of open source tools for cloud native security and observability — VMClarity, KubeClarity, and APIClarity. Read more about each project below.
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://docs.openclarity.io/img/footer-logos/OC_logo_H_1C_white.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://docs.openclarity.io/img/color-logo/logo.svg">
+  <img alt="Openclarity logo" src="https://docs.openclarity.io/img/color-logo/logo.svg" width="50%">
+</picture>
 
-## [VMClarity](https://github.com/openclarity/vmclarity)
+Openclarity is an open-source platform to enhance security and observability of cloud-native applications and infrastructure.
```
Comment (fekete-robert) [inline profile/README.md:10]:
```suggestion
Openclarity is an open source platform to enhance security and observability of cloud-native applications and infrastructure.
```

**Review by lgecse — CHANGES_REQUESTED** [review]

**Review by lgecse — APPROVED** [review]

**Comment by lgecse** [comment]
> @Tehsmash / @lgecse could either of you confirm that these project renames are in-line with broader plans to unify OpenClarity? Are we also changing the official casing from `OpenClarity` to `Openclarity` ?
> 
> (ref: [openclarity/vmclarity#869 (review)](https://github.com/openclarity/vmclarity/pull/869#pullrequestreview-1704460130))

@lelia Thanks for bringing this up, I confirm we are **not** changing `OpenClarity` to `Openclarity`. This was only a misunderstanding.

**Review by lgecse — APPROVED** [review]

File: profile/README.md [hunk 4]
```diff
@@ -1,11 +1,24 @@
 
-# [OpenClarity](https://openclarity.io)
+# [Openclarity](https://openclarity.io)
```
Comment (lgecse) [inline profile/README.md:2]: Please keep the OpenClarity naming, and make the whole file consistent.

File: profile/README.md [hunk 5]
```diff
@@ -1,11 +1,24 @@
...
-OpenClarity is a suite of open source tools for cloud native security and observability — VMClarity, KubeClarity, and APIClarity. Read more about each project below.
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://docs.openclarity.io/img/footer-logos/OC_logo_H_1C_white.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://docs.openclarity.io/img/color-logo/logo.svg">
+  <img alt="Openclarity logo" src="https://docs.openclarity.io/img/color-logo/logo.svg" width="50%">
+</picture>
 
-## [VMClarity](https://github.com/openclarity/vmclarity)
+Openclarity is an open source platform to enhance security and observability of cloud-native applications and infrastructure.
+Openclarity delivers:
```
Comment (lgecse) [inline profile/README.md:11]: Please keep the OpenClarity naming, and make the whole file consistent.

File: profile/README.md [hunk 6]
```diff
@@ -1,11 +1,24 @@
...
 
-VMClarity is an open source tool for agentless detection and management of Virtual Machine Software Bill Of Materials (SBOM) and security threats such as vulnerabilities, exploits, malware, rootkits, misconfigurations and leaked secrets.
+- Agent-less detection and managements of vulnerabilities, exploits, malware and misconfigurations for VMs and Container Images
+- Capabilities for runtime scans of Kubernetes and CI/CD pipelines
+- Comprehensive API security for internal and third-party APIs
+
+Openclarity is a suite of open source tools for cloud native security and observability — VM Security, Kubernetes Security, and API Security. Read more about each project below.
```
Comment (lgecse) [inline profile/README.md:17]: Please keep the OpenClarity naming, and make the whole file consistent.

**Review by lgecse — CHANGES_REQUESTED** [review]
Please keep the OpenClarity naming, and make the whole file consistent.

File: profile/README.md (hunk 5 above)
Comment (fekete-robert) [inline profile/README.md:11]:
```suggestion
[OpenClarity](https://openclarity.io) is an open source platform to enhance security and observability of cloud-native applications and infrastructure.
OpenClarity delivers:
```

File: profile/README.md (hunk 6 above)
Comment (fekete-robert) [inline profile/README.md:17]:
```suggestion
OpenClarity is a suite of open source tools for cloud native security and observability — VM Security, Kubernetes Security, and API Security. Read more about each project below.
```

File: profile/README.md (hunk 4 above)
Comment (fekete-robert) [inline profile/README.md:2]:
```suggestion
# [OpenClarity](https://openclarity.io)
```

File: profile/README.md [hunk 7]
```diff
@@ -1,11 +1,24 @@
 
-# [OpenClarity](https://openclarity.io)
+# [Openclarity](https://openclarity.io)
 
-OpenClarity is a suite of open source tools for cloud native security and observability — VMClarity, KubeClarity, and APIClarity. Read more about each project below.
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://docs.openclarity.io/img/footer-logos/OC_logo_H_1C_white.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://docs.openclarity.io/img/color-logo/logo.svg">
+  <img alt="Openclarity logo" src="https://docs.openclarity.io/img/color-logo/logo.svg" width="50%">
```
Comment (fekete-robert) [inline profile/README.md:7]:
```suggestion
  <img alt="OpenClarity logo" src="https://docs.openclarity.io/img/color-logo/logo.svg" width="50%">
```

**Review by lgecse — APPROVED** [review]

File: profile/README.md [hunk 8]
```diff
@@ -1,11 +1,24 @@
 
 # [OpenClarity](https://openclarity.io)
 
-OpenClarity is a suite of open source tools for cloud native security and observability — VMClarity, KubeClarity, and APIClarity. Read more about each project below.
+<picture>
```
Comment (justaugustus) [inline profile/README.md:4]:
```suggestion

<picture>
```

File: profile/README.md [hunk 9]
```diff
@@ -1,11 +1,24 @@
...
+  <source media="(prefers-color-scheme: dark)" srcset="https://docs.openclarity.io/img/footer-logos/OC_logo_H_1C_white.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://docs.openclarity.io/img/color-logo/logo.svg">
+  <img alt="OpenClarity logo" src="https://docs.openclarity.io/img/color-logo/logo.svg" width="50%">
+</picture>
 
-## [VMClarity](https://github.com/openclarity/vmclarity)
+[OpenClarity](https://openclarity.io) is an open source platform to enhance security and observability of cloud-native applications and infrastructure.
```
Comment (justaugustus) [inline profile/README.md:10]:
```suggestion
[OpenClarity](https://openclarity.io) is a suite of open source projects built to enhance the security and observability of cloud native applications and infrastructure.
```

***Not a platform

File: profile/README.md [hunk 10]
```diff
@@ -1,11 +1,24 @@
...
+OpenClarity delivers:
```
Comment (justaugustus) [inline profile/README.md:11]:
```suggestion

OpenClarity delivers:
```

File: profile/README.md [hunk 11]
```diff
@@ -1,11 +1,24 @@
...
 
-VMClarity is an open source tool for agentless detection and management of Virtual Machine Software Bill Of Materials (SBOM) and security threats such as vulnerabilities, exploits, malware, rootkits, misconfigurations and leaked secrets.
+- Agent-less detection and managements of vulnerabilities, exploits, malware and misconfigurations for VMs and Container Images
```
Comment (justaugustus) [inline profile/README.md:13]:
```suggestion
- Agentless detection and managements of vulnerabilities, exploits, malware and misconfigurations for virtual machines and container images
```

File: profile/README.md [hunk 12]
```diff
@@ -1,11 +1,24 @@
...
+- Capabilities for runtime scans of Kubernetes and CI/CD pipelines
+- Comprehensive API security for internal and third-party APIs
+
+OpenClarity is a suite of open source tools for cloud native security and observability — VM Security, Kubernetes Security, and API Security. Read more about each project below.
```
Comment (justaugustus) [inline profile/README.md:17]:
```suggestion
```

This repeats effectively repeats the opening line of the readme.

File: profile/README.md [hunk 13]
```diff
@@ -31,9 +44,9 @@ Key Capabilities:
 - Comprehensive dashboard for SBOM analysis
 - Pluggable architecture
 
-## [APIClarity](https://github.com/openclarity/apiclarity)
+## API Security
 
-APIClarity is an open source tool for APIs that helps you visualize and identify potential risks around API usage in your cloud native environments. It helps build the OpenAPI specifications for all APIs in your environment, then helps track drift, shadow or zombie usage for those APIs. You can then use this information to build your application security posture.
+[APIClarity](https://github.com/openclarity/apiclarity) is an open source tool for APIs that helps you visualize and identify potential risks around API usage in your cloud native environments. It helps build the OpenAPI specifications for all APIs in your environment, then helps track drift, shadow or zombie usage for those APIs. You can then use this information to build your application security posture.
```
Comment (justaugustus) [inline profile/README.md:49]:
```suggestion
[APIClarity](https://github.com/openclarity/apiclarity) is a tool that helps you visualize and identify potential risks around API usage in your cloud native environments. It helps build the OpenAPI specifications for all APIs in your environment, then helps track drift, shadow or zombie usage for those APIs. You can then use this information to build your application security posture.
```

File: profile/README.md [hunk 14]
```diff
@@ -1,11 +1,24 @@
...
-VMClarity is an open source tool for agentless detection and management of Virtual Machine Software Bill Of Materials (SBOM) and security threats such as vulnerabilities, exploits, malware, rootkits, misconfigurations and leaked secrets.
+- Agent-less detection and managements of vulnerabilities, exploits, malware and misconfigurations for VMs and Container Images
+- Capabilities for runtime scans of Kubernetes and CI/CD pipelines
+- Comprehensive API security for internal and third-party APIs
+
+OpenClarity is a suite of open source tools for cloud native security and observability — VM Security, Kubernetes Security, and API Security. Read more about each project below.
+
+## VM Security
+
+[VMClarity](https://github.com/openclarity/vmclarity) is an open source tool for agentless detection and management of Virtual Machine Software Bill Of Materials (SBOM) and security threats such as vulnerabilities, exploits, malware, rootkits, misconfigurations and leaked secrets.
```
Comment (justaugustus) [inline profile/README.md:21]:
```suggestion
[VMClarity](https://github.com/openclarity/vmclarity) is a tool for agentless detection and management of virtual machine Software Bill Of Materials (SBOM) and security threats such as vulnerabilities, exploits, malware, rootkits, misconfigurations, and leaked secrets.
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
Adding some minor copyedits.
I'll apply the suggestions and merge this. Feel free to take any outstanding reviews as follow-up PRs.

**Review by justaugustus — APPROVED** [review]

**Pull request merged**